sr_hand_health_report/src/sr_hand_health_report/tactile_check.py

```diff
@@ -27,21 +27,43 @@ class TactileCheck(SrHealthReportCheck):
 
     _REASONABLE_RANGE = {"pst": [200, 1200], "bt_sp": [1800, 2400], "bt_2sp": [1800, 2400]}
     _FINGERS = ("FF", "MF", "RF", "LF", "TH")
+    TIMEOUT = 2
     PASSED_THRESHOLDS = {'connected': True, 'reasonable': True}
 
     def __init__(self, hand_side):
+        """
+            Initialize the TactileCheck object
+            @param hand_side: String indicating the side
+        """
         super().__init__(hand_side, self._FINGERS)
+        self._name = "Tactile"
         self._topic_name = f"/{self._hand_prefix}/tactile"
-        self._pass_conditions = {'connected': True, 'reasonable': True}
+        self._result = {'tactile': {}}
+
         try:
-            self._serial = rospy.get_param(f"/sr_hand_robot/{self._hand_prefix}/hand_serial")
+            serials = rospy.get_param("/diagnostic_aggregator/load_diagnostic_analyzer/hand_serials_list")
+            if len(serials) == 2:
+                self._serial = serials[0] if self._hand_prefix == 'rh' else serials[1]
+            else:
+                self._serial = serials[0]
         except KeyError:
+            pass
+
+        try:
+            self._serial
+        except Exception:
             rospy.logerr("No hand detected!")
             sys.exit(1)
+
         self._expected_tactile_type = self.get_expected_tactile_type()
         self._topic_type_string = None
+        self._fingers_to_test = self.get_fingertips_from_config_file()
 
     def get_expected_tactile_type(self):
+        """
+            Checks the expected type from general_info.yaml file
+            @return: str expected fingertype
```

Comment:
```suggestion
            @return: str expected finger type
```

Reply:
What is a finger type? Do you mean finger name or prefix? Or do you mean tactile sensor type? Can you clarify this?